chroma-manager/chroma_api/command.py

```diff
@@ -109,15 +110,19 @@ def dismiss_all(self, request, **kwargs):
 
         return http.HttpNoContent()
 
-    def obj_create(self, bundle, request = None, **kwargs):
+    @validate
+    def obj_create(self, bundle, **kwargs):
+        request = bundle.request
+
         for job in bundle.data['jobs']:
             # FIXME: HYD-1367: This is a hack to work around the inability of
             # the Job class to handle m2m references properly, serializing hosts
             # to a list of IDs understood by the HostListMixin class
             if 'hosts' in job['args']:
                 job_ids = []
                 for uri in job['args']['hosts']:
-                    job_ids.append(HostResource().get_via_uri(uri).id)
+                    job_ids.append(HostResource().get_via_uri(
+                        uri, bundle.request).id)
```

Comment:
I was using a python auto-formatting tool that did this, think it was autopep8.

Ended up reverting most of them, as it made the diff very noisy.